site/docs/contribute/modules.md

```diff
@@ -2,20 +2,48 @@
 
 This page describes what must be provided when contributing a [module](../concepts/modules.md).
 
+A module is packaged as a Helm chart that the control plane can install to a workload's data plane.
+
+To first make a module available to the control plane, it must be registered by applying a FybrikModule custom resource.
+
+After that, a FybrikApplication can be submitted.  
+The FybrikApplication controller chooses the best modules to accommodate the data needs requested in the FybrikApplication, and an instace of each selected module is deployed.  
+
+Each module is deployed using its Helm chart, and vaules from the FybrikApplication are passed to the module using the values.yaml of the helm chart, so that the module has the needed configuration paramters.
```

Comment:
1. Module does not get values from the FybrikApplication. 
2. Values.yaml of the Helm are not used by Fybrik. 
3. Here is the place to provide the structure of values passed to the module in memory.

Reply:
I changed it to -

each module is deployed using its Helm chart.
Fybrik passes the needed configuration parameters to the module using the [Helm mechanism for passing values](https://helm.sh/docs/chart_template_guide/values_files/).

I am not sure a generated doc is needed, as the values are interpreted as .values anyway by helm.

@shlomitk1 do you think it is OK this way?

Reply:
The module developers need to know what they get as .Values structure. Since it is specific to Fybrik, we need to detail this structure.

Reply:
Added a reference to the place where the values are shown as we discussed